### ISSUE TITLE
Only able to stream outside of network 

### ISSUE THREAD

**Issue by tumekee** [homebridge-plugins/homebridge-camera-ffmpeg#443]
If i am connected to 4G or internet outside of my network i can live stream just fine.  If i am connected locally vai LAN or WIFI ( not that it should matter but tested both anyway )  i can get the still screenshot but not the live stream which is very odd. Im using Ubuntu server 18.4 with the firewall is turned off. Its almost like when the live stream is going via Apple TV its working , i dont know how live streams are handled locally to work out whats going on. any ideas ?

**Comment by driskill** [comment]
@tumekee same for me. I'm on macOS 10.15.4 , homebridge-camera-ffmpeg v0.1.15. On local net I can see the camera try to return data to the device but ends up with no response. Off home network on LTE, works fine. Don't know if it was my last macOS or ffmpeg update that broke it.

**Comment by alexndrs** [comment]
Same problem. Any fixes? I’m on Windows.

**Comment by ciavez** [comment]
I have the exact opposite problem... it works on LAN but not outside home

**Comment by tumekee** [comment]
@ciavez so you have a different problem.

**Comment by alexndrs** [comment]
My problem got solved changing operating system. I was running homebridge on Windows. Then changes to MacOSX but still same homebridge setup.

**Comment by meluvalli** [comment]
I'm also having this problem now.  I'm on Windows.

Started after I upgraded HomeBridge to latest version (1.0.4)  This is a new issue as my old version that I was running before (0.48 I think) didn't have this problem.

**Comment by alexndrs** [comment]
@meluvalli Tried to rollback but It didn’t work. It might be something with Windows update. The only solution for you I can figure out is to emulate for example Ubuntu on Windows with virtual box.

**Comment by meluvalli** [comment]
@alexndrs I backed up my complete homebridge directory before doing anything.

If I put back my old homebridge directory, it works.  So, not a Windows Update thing.
I played with it more just now, and I could have sworn I tried this earlier, but maybe I didn't...  Anyhow, I rolled back JUST homebridge-camera-ffmpeg back to v0.1.15 and it's now working.  I'm still running the latest NPM and HomeBridge.

So, this is what I'm running and it's now working:
Node.js: v14.2.0
NPM: v6.16.4
HomeBridge: v1.0.4
Homebridge-camera-ffmpeg: v0.1.15

**Comment by thoukydides** [comment]
Downgrading from homebridge-camera-ffmpeg 0.1.19 to 0.1.15 also fixed local streaming for me.

My working configuration is:
* OS: Ubuntu 18.04.4 LTS
* nodejs: 12.8.0
* npm: 6.14.5
* homebridge: 1.0.4
* homebridge-camera-ffmpeg: 0.1.19

**Comment by tumekee** [comment]
tried both @thoukydides and @meluvalli config as above and no luck on Ubuntu 18.04.4 and Ubuntu 20.04 its probably something simple...

**Comment by robertmehrer** [comment]
make sure bonjour isnt blocked on your local network

**Comment by tumekee** [comment]
so i worked it out after trying everything , it was because i have 2 NIC's on my server. using the "interfaceName": does nothing. disabled one of the NIC's and boom all fixed !
